src/main/java/com/damian/cryptoportfolio/presentation/web/HomeController.java

```diff
@@ -32,6 +32,7 @@ public String about(){
     @GetMapping("/login")
     public String login(){
 
+        
```

Comment:
Try to review before committing, this line doesnt add anything, just noise in the PR (Pull request)